pkg/ddl/backfilling_import_cloud.go

```diff
@@ -64,30 +67,32 @@ func newCloudImportExecutor(
 	}, nil
 }
 
-func (m *cloudImportExecutor) Init(ctx context.Context) error {
+func (c *cloudImportExecutor) Init(ctx context.Context) error {
```

Comment:
as a convention, please use the prefix of the word(s) that reflects its meaning as the receiver name, in this case, it's `Executor`

Reply:
https://github.com/pingcap/tidb/pull/60904/commits/a3e4c9b46be1824e02d6217e357b48dc32ee4e4d